pkg/virtctl/expose/expose_test.go

```diff
@@ -102,6 +102,12 @@ var _ = Describe("Expose", func() {
 				kubecli.GetKubevirtClientFromClientConfig = kubecli.GetMockKubevirtClientFromClientConfig
 			})
 		})
+		Context("With missing input parameters", func() {
+			It("should fail", func() {
+				cmd := tests.NewRepeatableVirtctlCommand(expose.COMMAND_EXPOSE)
+				Expect(cmd()).NotTo(BeNil())
```

Comment:
Nit: adding a `err := cmd()` and then expect `err` not to be nil would make this more readable.

Reply:
fixed